zookeeper-server/src/test/java/org/apache/zookeeper/test/ReadOnlyModeTest.java

```diff
@@ -198,6 +205,9 @@ public void testConnectionEvents() throws Exception {
      */
     @Test(timeout = 90000)
     public void testSessionEstablishment() throws Exception {
+        qu.enableLocalSession(true);
```

Comment:
this is not related to logback change? any info as to why we had to add this?

Reply:
Looks like few APIs have been changed due to which this code has to be added to pass this UT